taxcalc/functions.py

```diff
@@ -1049,22 +1050,20 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517, e37717,
     c05800 = _taxbc + c63200
 
     return (c62720, c60260, c63100, c60200, c60240, c60220, c60000,
-            c60130, c62730, _addamt, c62100, c04500,
+            c60130, c62730, _addamt, c62100,
             _amtsepadd, c62600, _agep, _ages, c62700, c62760,
             _alminc, _amtfei, c62780, c62900, c63000, c62740,
-            _ngamty, c62745, y62745, _tamt2, _amt5pc, _amt15pc,
+            _ngamty, c62745, _tamt2, _amt5pc, _amt15pc,
             _amt25pc, c62747, c62755, c62770, _amt20pc, c62800,
             c09600, _othtax, c05800, _cmbtp, c62100_everyone)
 
 
 @iterate_jit(nopython=True)
 def MUI(c00100, NIIT_thd, MARS, e00300, e00600, c01000, e02000, NIIT_trt,
-        NIIT):
-    if c00100 > NIIT_thd[MARS - 1]:
-        NIIT = NIIT_trt * min(e00300 + e00600 + max(0, c01000) +
-                              max(0, e02000), c00100 - NIIT_thd[MARS - 1])
-    else:
-        NIIT = 0
+        NIIT, e85070, e26270):
+    NIIT = NIIT_trt * min(e00300 + e00600 + max(0, c01000) + max(0, e02000 -
+                          e26270 + e85070), max(0, c00100 -
+                          NIIT_thd[MARS - 1]))
```

Comment:
The change that takes a max function on `c00100 - NIIT_thd[MARS - 1]` responds to the comment [line 19](https://www.irs.gov/pub/irs-pdf/f8960.pdf). And `e26270 ` corresponds to the [line 11](https://www.irs.gov/pub/irs-pdf/f8960.pdf) that should be deducted.